shared/src/schemas/Feedback.ts

```diff
@@ -0,0 +1,47 @@
+import * as yup from 'yup';
+
+import {SessionMode, SessionType} from './Session';
+import {transformTimestamp} from '../modelUtils/transform';
+
+export const FeedbackParamsSchema = yup.object().shape({
+  screen: yup.string(),
+  key: yup.string(),
```

Comment:
I can't see screen or key being sent from the client? But also I don't think it makes sense as we ask for feedback at the same places every time. So I believe it's safe to remove completely :)